src/sumo/wrapper/_new_auth.py

```diff
@@ -58,6 +58,11 @@ def get_token(self):
                 print(flow["message"])
                 result = self.msal.acquire_token_by_device_flow(flow)
 
+                if "error" in result:
+                    raise ValueError(
+                        "Failed to create device flow. Err: %s" % json.dumps(flow, indent=4)
```

Comment:
Flisespikring, men... "Failed to acquire token by device flow"?

Reply:
Stemmer!

Reply:
En litt rask copy/paste der..